### ISSUE TITLE
Interests input on new announcement page is not accessible

### ISSUE THREAD

**Issue by WilHall** [thoughtbot/constable#724]
The interests input on the new announcement page uses Selectize.js, and it is not friendly to screen readers for several reasons:
 - The input does not have a label
 - The placeholder is uninformative
 - Selectize.js is not very accessible:
   - When values are selected, the input reads as empty
   - Options are very difficult to navigate with a screen reader

![image](https://user-images.githubusercontent.com/342826/64829556-ce0ab980-d59a-11e9-81e8-01e79671186e.png)
